miso-web/src/main/webapp/scripts/indexDistance_ajax.js

```diff
@@ -36,14 +36,19 @@
         data: JSON.stringify(data)
       }).success(function(data) {
         var results = '';
-        if (!data || !data.length) {
+        if (data.warningEditDistance > data.errorEditDistance) {
+          results += 'Indices with edit distance of less than ' + Constants.warningEditDistance + ' are considered near-matches.\n';
+        }
+        results += 'Indices with edit distance of less than ' + Constants.errorEditDistance + ' are considered DUPLICATES.\n\n';
+        results += 'Indices were compared at the length of the shortest Index 1, which is ' + data.shortestIndexLength + ' bp.\n';
```

Comment:
Add an extra line break to separate from near/duplicate listing

Reply:
Updated.